mscore/palette.cpp

```diff
@@ -1163,6 +1163,45 @@ bool Palette::event(QEvent* ev)
       }
 
 //---------------------------------------------------------
+//   keyPressEvent
+//---------------------------------------------------------
+
+void PaletteScrollArea::keyPressEvent(QKeyEvent *event)
+      {
+      NewWizard* nw = mscore->getNewWizard();
```

Comment:
Maybe just a new property for the Palette ? like _selectable or _drawGrid ?
or we can reuse selectable?